website/addons/badges/model/badges.py

```diff
@@ -161,7 +161,7 @@ def verify(self, vtype='hosted'):
     @property
     def recipient(self):
         return {
-            'idenity': self.node._id,
+            'idenity': self.node._id,  # TODO: An unknown amount of code may depend on this typo
```

Comment:
Thanks; that was a quick note before I realized badges were irrelevant anyway. (this addon is turned off) :ghost: 

When it involves potentially outdated production data, I'm never quite sure whether a field is present at some level. :)